.github/workflows/push.yml

```diff
@@ -0,0 +1,90 @@
+name: Push Events
+
+on:
+  push:
+    branches:
+      - dev
+      - main
+
+permissions:
+  contents: write
+  id-token: write
+  deployments: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: Create release
+    runs-on: ubuntu-latest
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
+
+    steps:
+      - name: 🚀 Create Release
+        id: release
+        uses: agrc/release-composite-action@v1
+        with:
+          prerelease: ${{ github.ref_name == 'dev' }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          github-app-id: ${{ secrets.UGRC_RELEASE_BOT_APP_ID }}
+          github-app-key: ${{ secrets.UGRC_RELEASE_BOT_APP_KEY }}
+          github-app-name: ${{ secrets.UGRC_RELEASE_BOT_NAME }}
+          github-app-email: ${{ secrets.UGRC_RELEASE_BOT_EMAIL }}
+
+  deploy-dev:
+    name: Deploy to staging
+    runs-on: ubuntu-latest
+    needs: release
+    environment:
+      name: dev
+    if: github.ref_name == 'dev' && needs.release.outputs.release_created
+
+    steps:
+      - name: 🚀 Deploy
+        uses: agrc/cloud-run-docker-deploy-composite-action@dev
```

Comment:
**todo**: update this to v1 when it's ready

Reply:
I had this done on my local machine but forgot to push it. Thanks for the catch.